BBD-514: Dropdown Does Not Hide After Selecting A Collection Option

ashima-g opened the pull request:
Remove hover selectors and hide the list using the `visibility` property
instead of the `display` property to allow a slight delay for the click event
to be registered.

http://koen.kivits.com/articles/pure-css-menu/

Vic-Dev reviewed (changes requested):
Looks good. Tomorrow we will set you up to be able to attemp this fix against the client's site, just to make sure it fixes the problem for them.

jmbtutor committed:
Hide gb-custom-select list on item select

Create the "activated" flag and set it to true when the list should be
open. This flag controls a CSS class .gb-custom-select__activated to
control when the list is shown.

jmbtutor commented:
@Vic-Dev: Changed the implementation to be more robust by using a flag and class.

Vic-Dev commented on src/tags/select/gb-select.ts:
```diff
@@ -92,7 +94,7 @@ export class Select extends FluxTag<any> {
   }
 
   focusElement(e: Event & { preventUpdate: boolean }) {
```
Please remove this parameter (it's not being used) :)

Vic-Dev reviewed (changes requested):
Hey looks good. Please make that small fix, and rebase off develop if needed :)

ashima-g commented:
@Vic-Dev : The changes to gb-select have been made.
